engine/src/main/java/org/hibernate/validator/messageinterpolation/HibernateMessageInterpolatorContext.java

```diff
@@ -23,4 +27,22 @@
 	 * @return The currently validated root bean type.
 	 */
 	Class<?> getRootBeanType();
+
+	/**
+	 * @return the message parameters added to this context for interpolation
+	 *
+	 * @since 6.0
+	 */
+	default Map<String, Object> getMessageParameters() {
```

Comment:
Why did you decide to make this a default method? The only implementation is our own one.